src/Primrose/BrowserEnvironment.js

```diff
@@ -366,16 +367,11 @@ export default class BrowserEnvironment extends EventDispatcher {
     };
 
     const doPicking = () => {
-      updateAll();
+      updateAll();  
       this.input.resolvePicking(this.scene);
     };
 
-    const moveGround = () => {
```

Comment:
Oh! We need that line. Everything else about your PR here is right, though.

There is another issue, #123, that--if it had been addressed already--would mean that this line could be deleted. But that's not the case yet. I guess I could have broken up issue #122 and #123 to not reference each other and just waited for them to be addressed before creating a new issue for deleting this `moveGround` function.